### DISCUSSION TITLE
Admob support?

### DISCUSSION THREAD

**Discussion by kurgandk** [CommunityToolkit/Maui#1709] [New Feature Discussions]
Is there any (relatively simple) way to implement an Admob banner ad in an app created with CTM? (ie: no  XAML)

**Comment by bijington** [comment]
We don't provide support for AdMob. In the past I have used this plugin though: https://github.com/marcojak/MTAdmob

**Comment by kurgandk** [comment]
But isn't that only supporting xaml based apps?

**Reply by bijington** [reply]
The examples you see might be XAML based but anything you can do in XAML you can do in C#. After all XAML compiles down to C# in the build process 😉

**Comment by kurgandk** [comment]
Ok I'll give it a shot, thanks
